identity-did/src/document/core_document.rs

```diff
@@ -842,6 +842,18 @@ where
   }
 }
 
+impl<D, T, U, V> Default for CoreDocument<D, T, U, V>
+where
+  D: DID + KeyComparable + Serialize,
+  T: Serialize + std::default::Default,
+  U: Serialize,
+  V: Serialize,
+{
+  fn default() -> Self {
+    CoreDocument::builder(Default::default()).build().unwrap()
+  }
+}
```

Comment:
This panics 100% of the time because the builder lacks an `id`. Always avoid `unwrap` if possible. We do not need a default implementation here so this can be safely removed/

Reply:
I left this a placeholder just so I can find the best way to avoid a document clone in resolver.rs.